client/src/components/ThemeText/ThemeText.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ThemeText.css";
+
+export const HeaderText = ({ data }) => {
```

Comment:
Since these components are the same except we just have different CSS for it, we can merge these two and make 1 component

Reply:
Since we'll now have 1 component, we can conditionally render the CSS depending on the prop that we get.
For example, if we have a prop called `primary`:bool, then when true we use the `headerText` CSS class, otherwise we use the `subtitleText` CSS class.

e.g. 
```
<div className=`${primary ? headerText : subtitleText}`>
    {data}
</div> 
```

Reply:
Fixed (Nestor): combined both into `ThemeText` component.